### ISSUE TITLE
Cannot Update to 1.3.0

### ISSUE THREAD

**Issue by Helmethead2002** [homebridge/homebridge#2827]
**Describe Your Problem:**

Just to preface: I am not very sure about all of the little details of my setup, all of this is done in my spare time, so please forgive me if I don't understand a certain term or I forget to add something to this post, just tell me how to get it and I should be able to get it for you. 

So Homebridge 1.3.0 came out recently and naturally I wanted to update right away. I did the usual, opened my command line and started the process to update. `hb-service stop` then, `npm install -g homebridge@latest` then to wrap it all up, `hb-service start`. Once it starts back up I go to Config-UI and see that I am still running 1.1.7. I think I have tried everything, update Config-UI-X, restart hb-service, restart the computer. I know it's probably something super simple that I'm missing but I just can't figure it out. Thanks in advance for the help!

[**Link to my logs**](https://gist.github.com/Helmethead2002/949094678b951cc1510ad23bbe0ade97)

This is what I do/what I get when I update homebridge:
```
C:\Windows\System32>hb-service stop
Stopping Homebridge Service...
Homebridge Stopped

C:\Windows\System32>npm install -g homebridge@latest
C:\Program Files\nodejs\homebridge -> C:\Program Files\nodejs\node_modules\homebridge\bin\homebridge
+ homebridge@1.3.0
updated 1 package in 5.091s

C:\Windows\System32>hb-service start
Starting Homebridge Service...
Homebridge Started
```

[**Link to my config**](https://gist.github.com/Helmethead2002/c0a3b02b6c41ddbc15ee01e4c50f4a0b)

**Environment:**

* **Node.js Version**: v14.15.3
* **NPM Version**: 6.14.10
* **Homebridge Version**: Currently 1.1.7; trying to get to 1.3.0
* **Operating System**: Windows 10
* **Process Supervisor**: hb-service (I think, not too sure though)

**Comment by oznu** [comment]
Check you don't have another copy of Homebridge installed under:

```
C:\Users\xxxxxx\AppData\Roaming\npm
```

**Comment by Helmethead2002** [comment]
I looked there and found three files named homebridge: one was a .cmd file, one was a .ps1 (PowerShel), and the last one was  what I assumed to be a plain text file with no apparent file type. 
Doing a bit more digging I also have two Homebridge folders. One under ~\Roaming\npm\node_modules and its name is just homebridge; another one is under C:\Users and its name is .homebridge 
Something else that I just tried was using `npm uninstall -g homebridge` and when that process "successfully" completed homebridge was still able to start with 1.1.7 according to Config UI like nothing happened.

**Comment by oznu** [comment]
Try deleting the the one in `~\Roaming\npm\node_modules`, including the folder and and .cmd files.

**Comment by Helmethead2002** [comment]
I deleted the three files and homebridge started like normal, but when I deleted the folder under `~\Roming\npm\node_modules` homebridge said it couldn't find the homebridge installation. When I tried to reinstall homebridge to fix the problem it said the same thing. When looking at npm's output for the completed installation I noticed that it was installing to `C:\Program Files\nodejs\homebridge` and not `~\Roaming\npm\node_modules` I copied the files from one path to another and it started homebridge started in 1.3.1 perfectly! Thank you @oznu for your help! Now I gotta figure out how to get npm to install stuff in the right place
